cmake/toolchains/aarch64-linux-gnu.cmake

```diff
@@ -0,0 +1,30 @@
+set(CMAKE_SYSTEM_NAME Linux)
+set(CMAKE_SYSTEM_PROCESSOR aarch64)
+
+set(CMAKE_C_COMPILER "aarch64-linux-gnu-gcc")
+set(CMAKE_CXX_COMPILER "aarch64-linux-gnu-g++")
+set(CMAKE_LINKER "aarch64-linux-gnu-ld")
+
+set(CMAKE_FIND_ROOT_PATH_MODE_PROGRAM NEVER)
+set(CMAKE_FIND_ROOT_PATH_MODE_LIBRARY ONLY)
+set(CMAKE_FIND_ROOT_PATH_MODE_INCLUDE ONLY)
+
+set(CMAKE_C_FLAGS "-march=armv8-a")
+set(CMAKE_CXX_FLAGS "-march=armv8-a")
+
+# cache flags
+set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS}" CACHE STRING "c flags")
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS}" CACHE STRING "c++ flags")
+
+set(CMAKE_INSTALL_PREFIX install)
```

Comment:
Line19 - Line30 has nothing to do with the toolchain. 
They are the options (don't know what BUILD_PERF_TESTS and BUILD_TESTS are for) defined by mmdeploy.
I think we'd better keep toolchain cmake as clear as possible

Reply:
done.